todo_app/data/session_items.py

```diff
@@ -0,0 +1,344 @@
+from flask import session
```

Comment:
You must not rely on using the flask session cookie. This currently breaks your integration test.

This is a blocker for this passing review.

Reply:
Could you give more clarity on what you mean by not using flask session cookie